python/mxnet/contrib/__init__.py

```diff
@@ -28,5 +28,5 @@
 from . import tensorboard
 
 from . import text
-
+from . import onnx
```

Comment:
What happens if the onnx-package is not present?

Reply:
This is the import for Mxnet's onnx package, which is why I have given the relative path.

Reply:
What happens if the onnx-package is not present?

Reply:
This import is for mxnet.contrib.onnx package which will ship as part of the code and not the 'onnx' package which we are installing in CI setup process. So package will always be present.
Following is the folder structure of the new mxnet.contrib.onnx module that is being added in this PR.
mxnet
      ....
      |-- contrib
      |           |-- onnx
      |                     |-- _import
      |-- gluon
                  |-- ...
      ......
Please let me know if this wrong, and I have change something.

Reply:
Ah I understand, thanks for elaborating. Could there be a possible namespace clash?

Reply:
No, there is no namespace clash. The import module in onnx folder is name '_import'.

Reply:
Ok, thanks